### PR TITLE
Display map

### PR DESCRIPTION
**Pull request by RobertoCarrilloAvila**
# Description

## Preview:
https://github.com/RobertoCarrilloAvila/target/assets/16627969/30977e90-85b6-411c-944b-79519710649a

### CODE REVIEW

File: src/components/Chat/Chat.scss [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  display: flex;
+  flex-direction: column;
+
+  &__container {
+    margin: 0;
+    padding: 0  10px;
+  }
+
+  &__title {
+    color: shared.$black-color;
```
[the remark this answers is not in the document]
Comment (RobertoCarrilloAvila) [reply src/components/Chat/Chat.scss:13]: I'll create a new PR with the linter, I'm doindto include in the CI

File: src/components/Home/Home.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    Chat: chatComponent,
+  }
+
+  return (
+    <div className="home">
+      <div className="home__container">
+        <Navbar className="white" />
+
+        { components[displayedComponent]() }
+      </div>
```
[the remark this answers is not in the document]
Comment (RobertoCarrilloAvila) [reply src/components/Home/Home.js:31]: Nice! 👍

File: src/App.js [hunk 3]
```diff
@@ -2,12 +2,13 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 
 import PublicPaths from "components/Constants/PublicPaths";
 import PrivatePaths from "components/Constants/PrivatePaths";
+import PrivateRoute from "components/PrivateRoute/PrivateRoute";
```
[the remark this answers is not in the document]
Comment (RobertoCarrilloAvila) [reply src/App.js]: @afmicc  Do you mean something like this?
```js
import { PrivatePaths, PublicPaths } from "components/Constants/Paths";
```

[the remark this answers is not in the document]
Comment (afmicc) [reply src/App.js]: yes

**Review by afmicc — APPROVED** [review]

**Pull request merged**